hack/make-rules/test-cmd-util.sh

```diff
@@ -4669,3 +4669,257 @@ runTests() {
     exit 1
   fi
 }
+
+run_initializer_tests() {
+  set -o nounset
+  set -o errexit
+
+  create_and_use_new_namespace
+  kube::log::status "Testing --include-uninitialized"
+  ### Create the Initializer
+  kubectl create -f hack/testdata/initializer-configuration.yaml 2>&1 "${kube_flags[@]}"
```

Comment:
why do you need this if you're specifying a non-empty `pending` list in the pod manifest?